openlibrary/plugins/openlibrary/api.py

```diff
@@ -453,7 +453,11 @@ def GET(self):
         return json.dumps(metadata)
 
 
-class patron_observations(delegate.page):
+class patrons_observations(delegate.page):
+    """"
+    Fetches a patron's observations for a work, requires auth, intended
+    to be used internally to power the My Books Page & books pages modal
+    """"
```

Comment:
```suggestion
    """
    Fetches a patron's observations for a work, requires auth, intended
    to be used internally to power the My Books Page & books pages modal
    """
```
Remove extra double-quote symbols.